manifests/0000_70_cluster-network-operator_01_credentialsrequest.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
 spec:
+  serviceAccountNames:
```

Comment:
Is this the right file?
`cloud-network-config-controller` credential requests are stored in `manifests/02-cncc-credentials.yaml` and already have the `serviceAccountNames` set.

Reply:
Good catch, thanks